Gio-2.0.gir

```diff
@@ -74244,6 +74234,19 @@ set, for convenience.</doc>
           </parameter>
         </parameters>
       </method>
+      <method name="set_static_name" c:identifier="g_task_set_static_name">
```

Comment:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/368

And the next one, just when we fixed the previous one :)